src/views/services/list/components/ServiceLocation.tsx

```diff
@@ -35,11 +36,7 @@ const ServiceLocation: FC<ServiceLocationProps> = ({ s }) => {
       return (
         <>
           {_.map(s.status.loadBalancer.ingress, (ingress, i) => {
-            return (
-              <div className="co-truncate co-select-to-copy" key={i}>
-                {ingress.hostname || ingress.ip || '-'}
-              </div>
-            );
+            return <HostDetails key={i}>{ingress.hostname || ingress.ip || '-'}</HostDetails>;
```

Comment:
key shouldnt be index, but something like IP or hostname , please apply it to all things below

Reply:
Done!